### PR TITLE
test: updated tests by replacing external APIs

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
replaced some external APIs with ted APIs

## Summary by CodeRabbit

- **Bug Fixes**
	- Removed assertions for `offsetHeight` and `offsetWidth` in the test case related to a fixed canvas size being visible when height is selected as Fixed.
	- Updated URL references in the test script from "https://randomuser.me" to "http://host.docker.internal:8000" for image sources.
	- Updated API endpoints for two requests in the `API_Bugs_Spec.js` file.
	- Updated image URLs in JSON files for fixtures related to widgets in the client application.
- **Tests**
	- Updated API endpoint URLs in test scripts to enhance reliability and internal consistency.
	- Modified image source URLs in various test fixtures to align with updated internal hosting.
- **Documentation**
	- Updated source URLs in iframe configurations to reflect new internal hosting paths.

> [!WARNING]
> The provided command lacks proper tags. Please modify PR body, specifying the tags you want to include or use `/ok-to-test tags="@tag.All"` to run all specs.
> Explore the tags documentation [here](https://www.notion.so/appsmith/Ok-to-test-With-Tags-7c0fc64d4efb4afebf53348cd6252918)

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts [hunk 1]
```diff
@@ -137,8 +136,8 @@ describe(
       agHelper.AssertElementVisibility(locators._modal);
 
       //Verify that a fixed canvas size is visible when height is selected as Fixed
-      agHelper.AssertProperty(locators._modal, "offsetHeight", 1094);
-      agHelper.AssertProperty(locators._modal, "offsetWidth", 456);
+      // agHelper.AssertProperty(locators._modal, "offsetHeight", 1094);
```
Comment (ApekshaBhosale) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts:139]: let's remove these

File: app/client/cypress/limited-tests.txt [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # To run only limited tests - give the spec names in below format:
-cypress/e2e/Regression/ClientSide/Templates/Fork_Template_spec.js
-
+cypress/e2e/Regression/ClientSide/Widgets/Iframe/Iframe_onSrcDocChange_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/Iframe/Iframe_spec.js
+cypress/e2e/Regression/ServerSide/ApiTests/API_Bugs_Spec.js
+cypress/e2e/Regression/ClientSide/Widgets/ListV2/ListV2_SerververSide_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/TableV1/Table_Column_Resize_spec.js
+cypress/e2e/Regression/ClientSide/Widgets/TableV2/TableV2_Column_Resize_spec.js
```
Comment (ApekshaBhosale) [inline app/client/cypress/limited-tests.txt:7]: remember to revert limited test file before merge

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/cypress/fixtures/Listv2/Listv2JSObjects.json [hunk 3]
```diff
@@ -812,7 +812,7 @@
...
...host.docker.internal:8000/photo-1492529029602-33e53698f407.jpeg\",\n    \"country\": \"Australia\",\n    \"name\": \"oktest\"\n  },\n  {\n    \"id\": 566,\n    \"gender\": \"femal\",\n    \"latitude\": \"66\",\n    \"longitude\": \"33\",\n    \"dob\": \"2019-07-01T05:30:00Z\",\n    \"phone\": \"123456\",\n    \"email\": \"1026033274@qq.com\",\n    \"image\": \"\",\n    \"country\": \"china\",\n    \"name\": \"test1\"\n  },\n  {\n    \"id\": 570,\n    \"gender\": \"male\",\n    \"latitude\": null,\n    \"longitude\": null,\n    \"dob\": null,\n    \"phone\": null,\n    \"email\": \"balaji@appsmith.com\",\n    \"image\": null,\n    \"country\": null,\n    \"name\": \"Balaji\"\n  },\n  {\n    \"id\": 571,\n    \"gender\": \"male\",\n    \"latitude\": null,\n    \"longitude\": null,\n    \"dob\": null,\n    \"phone\": null,\n    \"email\": \"123\",\n    \"image\": null,\n    \"country\": null,\n    \"name\": \"AAAA\"\n  },\n  {\n    \"id\": 572,\n    \"gender\": \"male\",\n    \"latitude\": null,\n    \"longitude\": null,\n    \"dob\": null,\n    \"phone\": null,\n    \"email\": \"123@12.com\",\n    \"image\": null,\n    \"country\": null,\n    \"name\": \"balaji app viewer\"\n  }\n]",
```
Comment (trishaanand) [inline app/client/cypress/fixtures/Listv2/Listv2JSObjects.json:815]: Why was this change required? Doesn't seem related to external APIs

Comment (NandanAnantharamu) [reply app/client/cypress/fixtures/Listv2/Listv2JSObjects.json:815]: https://randomuser.me/api/portraits/med/women/39.jpg this was replaced by TED

Comment (trishaanand) [reply app/client/cypress/fixtures/Listv2/Listv2JSObjects.json:815]: Understood. Thanks!

File: app/client/cypress/limited-tests.txt (hunk 2 above)
Comment (NandanAnantharamu) [reply app/client/cypress/limited-tests.txt]: done

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Modal/Modal_spec.ts]: done

**Review by trishaanand — APPROVED** [review]

File: app/client/cypress/fixtures/IframeOnSrcDocChange.json [hunk 4]
```diff
@@ -60,7 +60,7 @@
                   "topRow": 6.0,
                   "bottomRow": 38.0,
                   "parentRowSpace": 10.0,
-                  "source": "https://www.example.com",
+                  "source": "http://host.docker.internal:8000/a.txt",
```
Comment (sharat87) [inline app/client/cypress/fixtures/IframeOnSrcDocChange.json:63]: I don't think this works.

1. On CI, port 8000 is bound to the Cloud services container, not TED.
2. The `a.txt` file doesn't exist in TED anymore.

Am I missing something? 🤔

**Pull request merged**